test/lib/browsers/playwright.ts

```diff
@@ -284,6 +284,10 @@ export class Playwright extends BrowserInterface {
     await page.goto(url, { waitUntil: 'load' })
   }
 
+  getPage() {
+    return page
+  }
```

Comment:
If you intended to introduce this as part of this changeset I think you'll also need to add it to the [`BrowserInterface`](https://github.com/vercel/next.js/blob/canary/test/lib/browsers/base.ts). Though maybe better to delete for now until we use it in a test so we can validate it's plumbed correctly

Reply:
yeah I don't need it until a later PR, I'll move it to that one